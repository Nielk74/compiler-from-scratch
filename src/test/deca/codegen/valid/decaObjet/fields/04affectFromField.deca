// Description:
//    Programme qui fait affecte les champs d'un objet dans des variables
//
// Resultats:
//    22
//    1.20000e+01
//
// Historique:
//    cree le 16/06/2022

class A {
    int x = 22;
    float y = 12;
}
{
    A a = new A();
    int x = a.x;
    float y = a.y;
    println(x);
    println(y);
}