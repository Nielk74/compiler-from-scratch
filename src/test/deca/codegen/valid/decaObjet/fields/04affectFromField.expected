22
1.20000e+01
